processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -100,6 +100,10 @@
     GENERAL_NOT_ALL_FORGED_CREATED( "Internal Error in creation of Forged Methods, it was expected all Forged Methods to finished with creation, but %s did not" ),
     GENERAL_NO_SUITABLE_CONSTRUCTOR( "%s does not have an accessible parameterless constructor." ),
 
+    BUILDER_MORE_THAN_ONE_BUILDER_CREATION_METHOD( "More than one builder creation method for \"%s\". Found methods: \"%s\". Builder will not be used. Consider implementing a custom BuilderProvider SPI.", Diagnostic.Kind.WARNING ),
+    BUILDER_MORE_THAN_ONE_BUILD_METHOD_DEFAULT_PATTERN( "More than one build method in \"%s\" for \"%s\". Found methods: \"%s\". Using default \"build\" method pattern. Consider to add @BuilderMapping in order to select the correct build method.", Diagnostic.Kind.ERROR ),
+    BUILDER_MORE_THAN_ONE_BUILD_METHOD_DEFINED_PATTERN( "More than one build method in \"%s\" for \"%s\". Found methods: \"%s\". Using defined \"%s\" method pattern.", Diagnostic.Kind.ERROR ),
```

Comment:
Same as above, maybe:

`More than one build method in \"%s\" for \"%s\". Found methods: \"%s\", but no method is named \"%s\" as defined in @BuilderMapping#builderFinisher.`